src/redux/books/books.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Books = () => (
+  <div>Books Section</div>
+);
+
```

Comment:
- [ ] Kindly place the Book component in the component folder the redux folder should only contain the redux reducers and store.

- [ ] Also, the Book component is expected to receive the book value as a prop and render the author and title.